src/Eppie.App/Eppie.App.Resources.Shared/Strings/en/Resources.resw

```diff
@@ -1314,4 +1322,8 @@ Subject: {3}
     <value>Mailboxes</value>
     <comment>Title for Mailboxes pane</comment>
   </data>  
+  <data name="UiScaleOptionSystemDefault" xml:space="preserve">
```

Comment:
Perhaps we should rename `UiScaleOptionSystemDefault` to `UIScaleOptionSystemDefault`